.yarnrc.yml

```diff
@@ -1,2 +1,7 @@
-yarnPath: ".yarn/releases/yarn-berry.cjs"
 nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
+    spec: "@yarnpkg/plugin-interactive-tools"
+
+yarnPath: .yarn/releases/yarn-sources.cjs
```

Comment:
Yarn geinstalleerd vanuit sources zodat `yarn npm audit` beschikbaar is. Zodra Yarn een release heeft gedaan zal ik 'm weer terugzetten op `berry`